SWIPs/swip-20.md

```diff
@@ -12,30 +12,30 @@ created: 2024-05-08
 
 ## Abstract
 
-Currently, one's initial capital, invested as stake, is permanently locked. This means that returns on this stake is purely based on the ROI of the redistribution game. The purpose of this SWIP is to change that: in extenuating circumstances, users will be allowed to withdraw their stakes.
+Currently, one's initial capital, invested as stake, is permanently locked, meaning that returns on this stake are solely based on the ROI of the redistribution game. The purpose of this SWIP is to change that: in extenuating circumstances, users will be allowed to withdraw their stakes.
 
 
 ## Objectives
 
-Should the effective price of a user's stake increase, they should be able to withdraw the appreciation, i.e., the amount that is no longer representative of the capital required to play and safeguard the system.
+Should the effective price of a user's stake increase, they should be able to withdraw the appreciation, i.e., the amount that no longer represents the capital required to play and safeguard the system.
 
 
 ## Context
 
-Currently, the smart contract can only increase a node's stake by sending extra collateral. Importantly, it does not allow node operators to  decrease their stake or any other way fully or partially withdraw their stake. This is what will be changed with this feature.
+Currently, the smart contract can only increase a node's stake by sending additional collateral. Importantly, it does not allow node operators to  decrease their stake or any other way to fully or partially withdraw their stake. This limitation is what will be changed with this feature.
 
 
 ## Specification
 
-This part describes how stakes should be created, modified, computed and withdrawn. As a first step, we need to obtain the *committed stake* of the user. This is set by user action, based on the following procedure:
+This section describes how stakes should be created, modified, computed, and withdrawn. As a first step, we need to determine the *committed stake* of the user. This is set by user action, based on the following procedure:
 
 ### The stake endpoint
 1.   The contract accepts an extra argument, `committedStake`, which sets the committed stake.
-2.  The contract asks the price oracle for the unit price of storage -- let us denote it with the symbol `unitPrice`.
-3.  The contract then adds whatever amount is sent with the transaction to the  stake entry for the address of `txorigins`. We can call this sum the *potential stake balance*, and denote it with the symbol `potentialStakeBalance`.
+2.  The contract asks the price oracle for the unit price of storage -- denoted here as `unitPrice`.
+3.  The contract then adds any amount sent with the transaction to the stake entry for the address of `txorigins`. This sum can be referred to as the *potential stake balance*, denoted as `potentialStakeBalance`.
```

Comment:
txorigin without -s

Reply:
Yes, as Viktor said, txorigins should be changed to txorigin everywhere (this was probably my mistake originally). Otherwise, looks great!